test/modules/lighthouse_service.spec.js

```diff
@@ -560,12 +560,149 @@ describe('lighthouse_service api', () => {
       }
       mock.mockRejectedValue(response)
 
-      const result = await lighthouse.failDestinationPlate(form)
+      const result = await lighthouse.failDestinationPlateBeckman(form)
 
       const expected = { success: false, errors: ['There was an error'] }
 
       expect(mock).toHaveBeenCalledTimes(1)
       expect(result).toEqual(expected)
     })
   })
+
+  describe('#createDestinationPlateBiosero', () => {
```

Comment:
This file is getting long, it might be worth splitting these tests into the separate systems.

Reply:
yes ok